docs/helm.rst

```diff
@@ -26,7 +26,7 @@ You also need to install the Kanister controller
 
 .. substitution-code-block:: console
 
-   $ helm install --name myrelease --namespace kanister kanister/kanister-operator --set image.tag=|version|
+   $ helm install myrelease --namespace kanister kanister/kanister-operator --set image.tag=|version|
```

Comment:
`helm install` supports a `--create-namespace` option to create the release namespace, if it's missing. Unless there is an explicit separate command to create the namespace, we should add this option to all `helm install` examples.

Reply:
@kale-amruta thanks for the updates. Can you add the `--create-namespace` option to the other example code too?

Reply:
@ihcsim I have updated all the examples but have left applications under [these](https://docs.kanister.io/helm.html#kanister-enabled-applications) docs since these are stale and we are not really maintaining these helm charts in the kanister docs eg: https://docs.kanister.io/helm_instructions/mysql_instructions.html
We should probably remove these docs as we are using examples to install the applications

Reply:
Mind creating a `documentation` issue to track the task to remove those examples? Thanks.